pkg/manager/member/tikv_scaler_test.go

```diff
@@ -221,6 +222,9 @@ func TestTiKVScalerScaleIn(t *testing.T) {
 				store := &pdapi.StoreInfo{
 					Store: &pdapi.MetaStore{
 						StateName: v1alpha1.TiKVStateUp,
+						Store: &metapb.Store{
```

Comment:
Should also add tiflash stores to cover your code change?

Reply:
fixed